packages/manager/modules/bm-server-components/src/service-status/service-status.html

```diff
@@ -54,12 +54,13 @@
 
     <!--Monitoring options-->
     <oui-tile-definition
+        class="overide-flex"
         data-ng-if="$ctrl.isMonitoringOptionsAvailable"
         data-term="{{ :: 'dedicated_server_dashboard_monitoring' | translate }}"
     >
-        <oui-tile-description>
+        <oui-tile-description class="text-wrap">
```

Comment:
**nitpick:** I think this can be removed.
```suggestion
        <oui-tile-description>
```

Reply:
In my opinion we should keep this class to break between words instead of characters